### DISCUSSION TITLE
wsi-was-a-mistake/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#26] [Announcements]
# wsi-was-a-mistake/

if you disagree you’re wrong

https://www.supergoodcode.com/wsi-was-a-mistake/

**Comment by rijnhard** [comment]
I mean more meat in the sandwich would make for good noms.

**Comment by BasiqueEvangelist** [comment]
Windows System Interface :+1:

**Comment by EliasOfWaffle** [comment]
The next post will be about compositor running in zink drivers? the mesa3d gitlab look's have amount of fixes for egl to be merged in future

**Comment by qwertychouskie** [comment]
Not Writing Enough Long Blog Posts Is A Mistake

if you disagree you’re wrong
